apps/website/app/blog/[slug]/page.tsx

```
@@ -0,0 +1,89 @@
+        <div className="w-full max-w-4xl">
+          <header className="mb-8 text-center">
+            <h1 className="mb-4 text-5xl font-bold leading-tight text-primary">
+              {data.title}
+            </h1>
+            <p className="text-sm italic text-gray-500">
+              By {data.author} • {data.date}
+            </p>
+          </header>
+          <article
+            className="prose prose-lg lg:prose-xl prose-gray mx-auto leading-relaxed text-gray-700"
+            dangerouslySetInnerHTML={{ __html: contentHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)